imod/mf6/simulation.py

```diff
@@ -814,6 +814,17 @@ def dump(
                 toml_content[cls_name][key] = model_toml_path.relative_to(
                     directory
                 ).as_posix()
+            elif key in ["gwtgwf_exchanges","split_exchanges"]:
```

Comment:
We are seeing a lot of ~~if else statements~~ customn logic being added due to the exchange lists. It would be good to see if a refactor could simplify stuff. For instance we could add an ExchangeManager or ExchangeCollection object to hold the exchanges. This logic can then be moved to a dump method in that new class

Reply:
that's not a bad idea, but the exchanges are just packages, so then I would prefer to make a package list/collection class for that . Since a `Model` is also a list of packages with some extra features, I think that if we make  a `PackageList`  then `Model ` should inherit from it (and use the same dump and from_file methods). But a refactoring like that deserves its own issue.

Reply:
That is an interesting idea. I will create a separate issue for it and will approve this one